src/main/scala/castalia/JsonConverter.scala

```diff
@@ -0,0 +1,31 @@
+package castalia
+
+import java.io.FileNotFoundException
+import java.net.URL
+
+import spray.json._
+
+import scala.util.Try
+
+/**
+  * Created by jens on 13-12-15.
+  */
+object JsonConverter {
+  def parseJson[T: JsonReader](filename: String): T = {
+    val resource: URL = getClass.getResource("/" + filename)
+    resource match {
+      case url : URL => unmarshalJsonToClass[T](url) getOrElse
+        (throw new UnmarshalException(s"Type could not be unmarshelled from $url"))
+      case _ => throw new FileNotFoundException(filename)
```

Comment:
To make it extra clear you can also wrap it in an `Option` which will be `None` if the input it `null`.